packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/ModelDescription.java

```diff
@@ -29,10 +39,13 @@ static public ModelDescription fromModel(Model model) {
 					.setModelVersion(model.getHeader().getModelVersion())
 					.setExtractedFrom(model.getHeader().getExtractedFrom()));
 		}
-
 		return desc;
 	}
 
+	@Id
```

Comment:
In its current form, you can get rid of the `@Id` and `@GeneratedValue` annotations I think (tagging @kbirk here to be sure)  - this ID doesnt get added to the postgres database so these aren't autogenerated, and from the look of the code these are all being set off of their parent's ID in the block above?